modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -25,6 +25,47 @@
         </div>
       </AposContextMenuDialog>
     </bubble-menu>
+    <floating-menu
+      class="apos-rich-text-insert-menu" :should-show="showFloatingMenu"
+      :editor="editor" :tippy-options="{ duration: 100, zIndex: 2000 }"
+      v-if="editor"
+    >
+      <div class="apos-rich-text-insert-menu-heading">
+        {{ $t('apostrophe:richTextInsertMenuHeading') }}
+      </div>
+      <div
+        v-for="(item, index) in insert"
+        :key="`${item}-${index}`"
+        class="apos-rich-text-insert-menu-item"
+      >
+        <div class="apos-rich-text-insert-menu-icon">
+          <AposIndicator
+            :icon="insertMenu[item].icon"
+            :icon-size="35"
+            class="apos-button__icon"
+            fill-color="currentColor"
+            @click="activateInsertMenuItem(item, insertMenu[item])"
+          />
+          <component
+            v-if="item === activeInsertMenuComponent?.name"
+            :is="activeInsertMenuComponent.component"
+            :active="true"
+            :editor="editor"
+            :options="editorOptions"
+            @before-commands="removeSlash"
```

Comment:
would be nice to remove the slash when adding tables also... in another PR

Reply:
Fixed

Reply:
Fixed both issues. The one with the inline images was hard! But worth it